### PR TITLE
Bundles create enhancements

### PR DESCRIPTION
**Pull request by vanstee**
* [x] Add optional `--enable` flag (disabled will remain the default)
* [x] Add optional `--relay-groups` flag that accepts a list of group names
* [x] Explain all actions taken (Created bundle, added or failed to add bundle to groups) and then print full list of attributes for the bundle installed

### CODE REVIEW

**Comment by vanstee** [comment]
The tests have failed because I'm waiting to merge https://github.com/operable/cog/pull/550

File: lib/cogctl/actions/bundles/create.ex [hunk 1]
```diff
@@ -41,22 +47,74 @@ defmodule Cogctl.Actions.Bundles.Create do
...
+  defp create_bundle(endpoint, params, config) do
+    params = Map.put(params, "config", config)
+    Client.bundle_create(endpoint, params)
+  end
+
+  defp assign_to_relay_groups(endpoint, bundle, %{"relay-groups": relay_group_names}) do
+    relay_groups = String.split(relay_group_names, ",")
+
+    Enum.reduce_while(relay_groups, {:ok, []}, fn relay_group, {:ok, acc} ->
+      result = Client.relay_group_add_bundle(%{name: relay_group}, %{bundle: bundle.name}, endpoint)
```
Comment (mpeck) [inline lib/cogctl/actions/bundles/create.ex]: This function changed a bit with the updates to cog_api. Now it takes a list of bundles and add them all to the relay group.

`Client.relay_group_add_bundles(%{name: relay_group}, %{bundles: [bundle.name]}, endpoint)`

File: lib/cogctl/actions/bundles/create.ex [hunk 2]
```diff
@@ -27,12 +29,16 @@ defmodule Cogctl.Actions.Bundles.Create do
...
 
   def run(options, _args, _config, endpoint) do
     case convert_to_params(options, [file: :required,
-                                     templates: :required]) do
+                                     templates: :required,
+                                     enabled: :optional,
+                                     "relay-groups": :optional]) do
       {:ok, params} ->
         with_authentication(endpoint, &do_create(&1, params))
       :error ->
```
Comment (mpeck) [inline lib/cogctl/actions/bundles/create.ex]: We may have a conflict here. https://github.com/operable/cogctl/pull/49

`convert_to_params` returns an error with a list missing required args now.
`{:error, {:missing_params, <missing_params>}}`

File: lib/cogctl/actions/bundles/create.ex (hunk 1 above)
Comment (vanstee) [inline lib/cogctl/actions/bundles/create.ex]: Ah cool. Fixin'

File: lib/cogctl/actions/bundles/create.ex (hunk 2 above)
Comment (vanstee) [inline lib/cogctl/actions/bundles/create.ex]: Rebased cleanly. Thanks for the heads up.

**Pull request merged**